src/app/zap-templates/common/ClusterTestGeneration.js

```diff
@@ -117,6 +118,10 @@ function setDefaultTypeForCommand(test)
     test.commandName      = 'Write';
     test.isAttribute      = true;
     test.isWriteAttribute = true;
+    if ((kGroupId in test)) {
```

Comment:
Should probably add this to the "default" case too, but followup is OK for that.